.github/workflows/CI.yml

```
@@ -30,6 +30,15 @@ jobs:
         os:
```

```suggestion
        depwarn: ['']
        os:
```
Is this maybe needed for the new jobs to get created?

---

You could also just add a separate job, if you don't want to do this dance.

---

> Is this maybe needed for the new jobs to get created?

Yes that seems likely, you can use the `include` block for two things: add extra variables to existing (matching) jobs or to add extra jobs. Which one of these is used depends on whether all variables are already defined in the matrix definition, more details [here](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations).